src/templates/common/link/index.tsx

```diff
@@ -1,44 +0,0 @@
-import React from 'react'
```

Comment:
This is not used; all instances of <Link> are imported directly from 'next/link'